api/openai/list.go

```diff
@@ -15,14 +17,39 @@ func ListModelsEndpoint(loader *model.ModelLoader, cm *config.ConfigLoader) func
 		var mm map[string]interface{} = map[string]interface{}{}
 
 		dataModels := []OpenAIModel{}
-		for _, m := range models {
-			mm[m] = nil
-			dataModels = append(dataModels, OpenAIModel{ID: m, Object: "model"})
+
+		skipBinIfModelConfigured := false
```

Comment:
should we maybe keep this as a parameter as well like `filter` and default to false? I'm thinking because - in some cases can be useful to keep both around (for instance, benchmarking defaults and without prompts while still having the model configured)

Reply:
Sure thing - I was trying to keep things simple, but I'll go ahead and toss it on as a seperate query string parameter. I may end up changing the name of the variable / polarity of the if check, but the end result will be that the default behavior removes the duplicate entries and the override allows their inclusion.

Reply:
Tested as working with two seperate params on my mac.